package.json

```
@@ -34,7 +35,7 @@
   },
   "jest": {
     "setupFilesAfterEnv": [
-      "jest-extended/all"
+      "./testSetup.js"
```

```suggestion
      "jest-sorted",
      "jest-extended/all"
```

I don't think you'll need the testSetup.js